### PR TITLE
Add KubernetesTask scaffolding

### PR DESCRIPTION
**Pull request by nemacysts**
This sets up a basic skeleton of Tron's wrapper around a task_processing
task (and is very similar to the corresponding Mesos version: MesosTask)

### CODE REVIEW

**Comment by nemacysts** [comment]
(this is a draft until we have KubernetesTaskConfig available in task_proc)

File: tron/kubernetes.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            )
+            return
+
+        k8s_type = getattr(event, "platform_type", None)
+        self.log.info(f"Got event for task={event_id} (Kubernetes type={k8s_type}).")
+
+        try:
+            self.log_event_info(event=event)
+        except Exception:
+            self.log.exception(f"Unable to log event info for id={event_id}.")
```
Comment (lanceleih) [inline tron/kubernetes.py:91]: We can also add a print message of the exception, but that can be done later on.

**Review by lanceleih — APPROVED** [review]
LGTM

File: tron/kubernetes.py (hunk 1 above)
Comment (nemacysts) [reply tron/kubernetes.py:91]: log.exception will already do that :) (https://devdocs.io/python~3.6/library/logging#logging.exception)

**Pull request merged**